hathor/feature_activation/resources/feature.py

```diff
@@ -0,0 +1,144 @@
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+from typing import Optional
+
+from hathor.api_util import Resource, set_cors
+from hathor.cli.openapi_files.register import register_resource
+from hathor.feature_activation.feature import Feature
+from hathor.feature_activation.feature_service import FeatureService
+from hathor.feature_activation.model.feature_state import FeatureState
+from hathor.feature_activation.settings import Settings
+from hathor.transaction.storage import TransactionStorage
+from hathor.utils.api import Response
+
+
+@register_resource
+class FeatureResource(Resource):
+    __slots__ = ()
+
+    isLeaf = True
+
+    def __init__(
+        self,
+        *,
+        settings: Settings,
+        feature_service: FeatureService,
+        tx_storage: TransactionStorage
+    ) -> None:
+        super().__init__()
+        self._settings = settings
+        self._feature_service = feature_service
+        self.tx_storage = tx_storage
+
+    def render_GET(self, request):
+        request.setHeader(b'content-type', b'application/json; charset=utf-8')
+        set_cors(request, 'GET')
+
+        best_block = self.tx_storage.get_best_block()
+        bit_counts = best_block.get_feature_activation_bit_counts()
+        features = []
+
+        for feature, criteria in self._settings.features.items():
+            state = self._feature_service.get_state(block=best_block, feature=feature)
+            acceptance_count = bit_counts[criteria.bit]
+            acceptance = (
+                acceptance_count / self._settings.evaluation_interval if state is FeatureState.STARTED else None
+            )
+            threshold_count = (
+                criteria.threshold if criteria.threshold is not None else self._settings.default_threshold
+            )
+
+            feature_response = GetFeatureResponse(
+                name=feature,
+                state=state.name,
+                acceptance=acceptance,
+                threshold=threshold_count / self._settings.evaluation_interval,
+                start_height=criteria.start_height,
+                minimum_activation_height=criteria.minimum_activation_height,
+                timeout_height=criteria.timeout_height,
+                activate_on_timeout=criteria.activate_on_timeout,
+                version=criteria.version
+            )
+
+            features.append(feature_response)
+
+        response = GetFeaturesResponse(features=features)
```

Comment:
Add information about the block related to these features. I guess the block id and height are enough.

Reply:
Done in https://github.com/HathorNetwork/hathor-core/pull/634/commits/6a8d8ab3d82cbd99c98536abb421644cab724364

Reply:
Great! Thanks!